lib/transform/transformers.js

```diff
@@ -30,7 +30,10 @@ export function assets (asset) {
   transformedAsset.fields.file = reduce(
     asset.fields.file,
      (newFile, file, locale) => {
-       newFile[locale] = omit(file, 'url', 'details')
+       if (newFile[locale].uploadFrom) {
+         delete newFile[locale].uploadFrom
+       }
```

Comment:
if the asset was deleted from the upload api (because more than 24 hours have passed) then there's nothing else we can do. If the asset wasn't deleted then we would still import it.